testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/ResourceOwnerPasswordCredentialsGrantTest.java

```diff
@@ -133,25 +133,20 @@ public void addTestRealms(List<RealmRepresentation> testRealms) {
                 .password("password");
         realm.user(defaultUser);
 
-        userId = KeycloakModelUtils.generateId();
         UserRepresentation user = UserBuilder.create()
-                .id(userId)
```

Comment:
@hmlnarik In this line you are removing user id, is it generated automatically?

Reply:
Yes